helm/values-prod.yaml

```diff
@@ -75,10 +75,10 @@ microservice-chart:
     APPLICATIONINSIGHTS_CONNECTION_STRING: 'ai-p-connection-string'
     PDF_BLOB_STORAGE_CONN_STRING: 'pdf-blob-storage-p-connection-string'
```

Comment:
fix `PDF_BLOB_STORAGE_CONN_STRING` for PROD

Reply:
updated